vertx/src/main/java/module-info.java

```diff
@@ -15,6 +17,6 @@
             com.microsoft.gctoolkit.api;
 
     provides com.microsoft.gctoolkit.jvm.JavaVirtualMachine with
-            com.microsoft.gctoolkit.vertx.jvm.DefaultJavaVirtualMachine;
+            PreUnifiedJavaVirtualMachine;
```

Comment:
The convention for module-info is to use the fully qualified name rather than imports.